server/utils/hashing.ts

```diff
@@ -0,0 +1,9 @@
+import { hash, compare } from 'bcryptjs';
+
+export function hashPassword(password: string): Promise<string> {
+  return hash(password, 10);
+}
+
+export function comparePassword(password: string, hashedPassword: string): Promise<boolean> {
+  return compare(password, hashedPassword);
```

Comment:
No need for this file, All you need to do when compare is to call `bcrypt` compare method, So Why to have a separate function?